src/porting.cpp

```diff
@@ -353,14 +353,21 @@ bool setSystemPaths()
 		"Failed to get current executable path");
 	pathRemoveFile(buf, '\\');
 
+	std::string exepath(buf);
+
 	// Use ".\bin\.."
-	path_share = std::string(buf) + "\\..";
+	path_share = exepath + "\\..";
+	if (detectMSVCBuildDir(exepath)) {
+		// The msvc build dir schould normaly not present if properly installed,
+		// but its usefull for debugging.
+		path_share += DIR_DELIM "..";
+	}
 
-	// Use "C:\Documents and Settings\user\Application Data\<PROJECT_NAME>"
+	// Use "C:\Users\<user>\AppData\Roaming\<PROJECT_NAME_C>"
 	DWORD len = GetEnvironmentVariable("APPDATA", buf, sizeof(buf));
 	FATAL_ERROR_IF(len == 0 || len > sizeof(buf), "Failed to get APPDATA");
 
-	path_user = std::string(buf) + DIR_DELIM + PROJECT_NAME;
+	path_user = std::string(buf) + DIR_DELIM + PROJECT_NAME_C;
```

Comment:
Find the wrong one: 😉 
![grafik](https://user-images.githubusercontent.com/3142688/28752102-e4561e90-7517-11e7-95a6-defaea022dc3.png)